.github/workflows/release-automation.yaml

```diff
@@ -39,3 +39,20 @@ jobs:
     needs: upload-pkg-and-dependency-source-code-to-release
     secrets: inherit
     uses: ./.github/workflows/release-homebrew.yaml
+
+  build-and-test-finch-msi:
+    build-and-test-finch-pkg:
+    needs: get-latest-tag
+    uses: ./.github/workflows/build-and-test-msi.yaml
+    secrets: inherit
+    with:
+      ref_name: ${{ needs.get-latest-tag.outputs.tag }}
+  
+  upload-msi-to-release:
+      needs:
```

Comment:
Dense for this line is wrong

Reply:
Thanks, seems like I messed up some of the merge conflicts. Will push fixes after tests run.